examples/jsm/loaders/MaterialXLoader.js

```diff
@@ -35,42 +44,32 @@ class MXElement {
 
 // Ref: https://github.com/mrdoob/three.js/issues/24674
 
-const mx_add = ( in1, in2 = float( 0 ) ) => add( in1, in2 );
```

Comment:
I don't have sure if its is work, I added `mx_add` because in TSL we don't have default value for the second parameter like MaterialX have.

https://materialx.org/assets/MaterialX.v1.38.Spec.pdf

Reply:
I will address that now.  I will check the MaterialX spec for defaults and add them.

Reply:
I think we can leave these minor details for another PR; I just made some final updates.

Thanks for the great PR.